source/includes/list-table-shell-properties.rst

```diff
@@ -103,12 +103,16 @@
    * - ``redactHistory``
      - string
      - ``remove``
-     - Controls what information is recorded in the shell history.
+     - Controls what information is recorded in the ``mongosh`` command history, 
+       meaning what can be accessed via up and down keys and Ctrl+R while inside the shell. 
+
        Must be one of:
```

Comment:
[suggestion] I know you didn't make this change, but it might be clearer if this introductory text doesn't end with a preposition
```suggestion
       Must be one of the following values:
```